dm/pkg/binlog/event/util.go

```diff
@@ -354,6 +354,23 @@ func GetParserForStatusVars(statusVars []byte) (*parser.Parser, error) {
 	return parser2, err
 }
 
+// GetServerCollationByStatusVars gets server collation by binlog statusVars.
+func GetServerCollationByStatusVars(statusVars []byte) (string, error) {
+	vars, err := statusVarsToKV(statusVars)
+	b, ok := vars[QCharsetCode]
+
+	if !ok {
+		if err == nil {
+			// only happen when this is a dummy event generated by DM
+			err = fmt.Errorf("Q_CHARSET_CODE not found in status_vars %v", statusVars)
+		}
+		// mysql default 'latin1_swedish_ci'
+		return mysql.Collations[8], err
```

Comment:
```suggestion
		return 'latin1_swedish_ci', err
```
`8` is a inner constant in tidb that should not be rely on.  The proper way to do this is to use `mysql.Collations[mysql.CollationNames['latin1_swedish_ci']]`, thus why not `latin1_swedish_ci` directly.

Reply:
ok